sample/AspNetFullFrameworkSampleApp/AspNetFullFrameworkSampleApp.csproj

```diff
@@ -322,6 +216,44 @@
   <ItemGroup Condition="'$(OS)' != 'WINDOWS_NT'">
     <Content Include="LinuxReadme.md" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
```suggestion
  <ItemGroup Condition="'$(OS)' == 'WINDOWS_NT'">
```

Reply:
We need this, otherwise the references will show up on Linux and macOS - we only want this project to do anything on Windows.